### PR TITLE
Split web apps templates: form_entry_templates.html

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical Summary
Same idea as https://github.com/dimagi/commcare-hq/pull/34478 and related PRs. This is the last one.

### Safety story
Mechanical changes, local testing is appropriate.

### Automated test coverage

I don't think these templates are included in the cloudcare js tests.

### QA Plan

no

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script type="text/html" id="blank-entry-ko-template"></script>
```
Comment (MartinRiese) [inline corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html:1]: I suppose for consistencies sake that that is ok 🤔

**Review by MartinRiese — APPROVED** [review]

File: corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html:1]: Tell me more about what you don't like here?

(I don't like that it's a 3-line file, but not sure if you're talking about something else.)

Comment (MartinRiese) [reply corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html:1]: Yeah, that just seemed kind of silly. But it makes sense. I rather have that than a 700 line file

Comment (orangejenny) [reply corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html:1]: agreed on both points

**Pull request merged**